danger.gemspec

```diff
@@ -26,8 +26,13 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "colored2", "~> 3.1"
   spec.add_runtime_dependency "faraday", "~> 0.9"
   spec.add_runtime_dependency "faraday-http-cache", "~> 1.0"
+  if RUBY_VERSION >= "2.3.0"
```

Comment:
I thought dependencies where sorted alphabetically, so I put this conditional here, but then noticed `cork` is after `terminal-table`. I'm open to moving this somewhere else if it looks better.

Reply:
I'm not too precious on the order etc, this is totally fine

Looks like unfortunately this technique doesn't work though: https://stackoverflow.com/questions/39346334/manage-gem-dependencies-based-on-ruby-version

There might not even be a way to do this, so maybe we'll need to go to use 2.0 only? Does that mean forcing 2.3.0 only?

Reply:
Yes, kramdown >2 requires Ruby 2.3. kramdown-parser-gfm has dependencies that will fail to install with Ruby < 2.3...

Maybe Danger needs a major version update?
- keep requiring `"kramdown", "~> 1.5"` in 5.x
- switch to `"kramdown", "~> 2.0"`  with the addition of `"kramdown-parser-gfm, "~> 1.0"` in 6.x, with the added requirement of Ruby 2.3+

5.x can keep getting bugfixes for a few months to allow a smooth transition.